docs/projects/post-release.rst

```diff
@@ -0,0 +1,173 @@
+.. _projects-post-release:
+
+================================
+Post-Release Docs Update Process
+================================
```

Comment:
Hi Naomi,

when thinking about this again, I believe we should not publish that kind of specific information here. Can you remove this document `post-release.rst` completely?

With kind regards,
Andreas.